src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -475,7 +477,13 @@ private ResponseEntity storeDocuments(SubmissionKey submissionKey, List<Multipar
 
         MDC.put(Keys.MDC_EFILING_SUBMISSION_ID, submissionId.toString());
 
-        fromCacheSubmission.get().getFilingPackage().setRush(rushProcessingMapper.toRushProcessing(rush));
+        RushProcessing rushProcessing = rushProcessingMapper.toRushProcessing(rush);
+        //Set filenames values not present in mapper for done here
```

Comment:
I'm not entirely sure what this comment means. It might need to be reworded a little bit.